slides/src/content/directives.md

```diff
@@ -306,9 +295,9 @@ Notes:
 
 ## `NgSwitch` Directive
 
-- Very similar to a switch statement
-- Multiple components can be matched using `ngSwitchCase`
-- Since components are created or destroyed be aware of the costs in doing so.
+- Very similar to a switch statement <!-- .element: class="smaller-font" -->
+- Multiple components can be matched using <!-- .element: class="smaller-font" --> `ngSwitchCase`
+- Since components are created or destroyed be aware of the costs in doing so. <!-- .element: class="smaller-font" -->
```

Comment:
Instead of reducing size, I would just remove the first item in the list which is quite obvious. The text would be:

> - ~Very similar to a switch statement~
> - Multiple components can be matched using `ngSwitchCase`
> - Since components are created or destroyed be aware of the costs in doing so.